meta-lmp-base/recipes-kernel/linux/kmeta-linux-lmp-5.15.y.inc

```diff
@@ -1,4 +1,4 @@
 KERNEL_META_REPO ?= "git://github.com/foundriesio/lmp-kernel-cache.git"
 KERNEL_META_REPO_PROTOCOL ?= "https"
 KERNEL_META_BRANCH ?= "linux-v5.15.y"
-KERNEL_META_COMMIT ?= "8203249516013fc2fd292da5d204d802808868c0"
+KERNEL_META_COMMIT ?= "e91c579da9f7c3847cffd5720b13fb75216a7d38"
```

Comment:
Heh.. I also have a change to the commit in my `imx8mn-ddr4-evk` patchset

Reply:
let me know where you want to keep it.